samples/Build2019Demo.Unity/Assets/Generated.StateSynchronization.AssetCaches/Resources/Android/spectatorview.manifest.bytes

```diff
@@ -52,7 +52,6 @@ Assets:
 - Assets/Generated.StateSynchronization.AssetCaches/Resources/MaterialPropertyAssetCache.asset
 - Assets/Generated.StateSynchronization.AssetCaches/Resources/AudioMixerGroupAssetCache.asset
 - Assets/Generated.StateSynchronization.AssetCaches/Resources/TextureAssetCache.asset
-- Assets/Demo/PlaygroundButtons/Shaders/Rounded_Edge_Thick.shader
```

Comment:
@matthejo , any idea why when you generate bundles, you include this, but when I generate them, I exclude it?

Reply:
I think last time I committed, I was testing something out with loading a shader directly and had included it explicitly in the bundle through the Unity UI. It should not be there, yours is correct.

---
In reply to: [343311829](https://github.com/microsoft/MixedReality-SpectatorView/pull/270#discussion_r343311829) [](ancestors = 343311829)